config/internal/mcad/deployment.yaml.tmpl

```diff
@@ -40,6 +40,20 @@ spec:
           volumeMounts:
             - name: temp-vol
               mountPath: /tmp
+          livenessProbe:
+            httpGet:
+              path: /healthz
+              port: 8081
+            initialDelaySeconds: 30
+            timeoutSeconds: 5
+            periodSeconds: 5
+          readinessProbe:
+            httpGet:
+              path: /readyz
+              port: 8081
+            initialDelaySeconds: 30
```

Comment:
Isn't 30 seconds too much?
How fast does MCAD usually start?

Reply:
I am thinking whether there is actually any need for `initialDelaySeconds` for MCAD.
IMHO this is mainly leveraged for slowly starting containers when there is no need to run readiness/liveness probes too soon.
The image pulling doesn't count to the startup time, it should start once pod is provisioned.

Reply:
so we're in agreement here? there is no need for the `initialDelaySeconds` ?

Reply:
+1